client/console/src/main/java/org/apache/syncope/client/console/panels/UserDirectoryPanel.java

```diff
@@ -342,6 +343,23 @@ public void onClick(final AjaxRequestTarget target, final UserTO ignore) {
             }, ActionType.MANAGE_ACCOUNTS,
                     String.format("%s,%s,%s", StandardEntitlement.USER_READ, StandardEntitlement.USER_UPDATE,
                             StandardEntitlement.RESOURCE_GET_CONNOBJECT));
+
+            panel.add(new ActionLink<UserTO>() {
```

Comment:
Please ensure that this action is not available in other contexts e.g. when showing group members.